nerve/check_https_test.go

```diff
@@ -3,12 +3,22 @@ package nerve
 import (
 	"github.com/stretchr/testify/require"
 	"testing"
+	"net/http/httptest"
+	"net/http"
+	"fmt"
 )
 
 func TestHttpsDefault(t *testing.T) {
```

Comment:
you should also check when the service is not there, after all it's more interesting to check when it fail